addons/common/fnc_updatePerFrameHandlerDelay.sqf

```diff
@@ -0,0 +1,42 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_updatePerFrameHandlerDelay
+
+Description:
+    Updates the delay of an existing perFrameHandler.
+
+Parameters:
+    _handle   - The existing perFrameHandler's handle. <NUMBER>
```

Comment:
```suggestion
    _handle   - The existing perFrameHandler's handle. (optional, default: -1) <NUMBER>
```

Reply:
while technically optional, not practical